compiler/modules/comm.diagnostic/diagnostic.bal

```diff
@@ -126,17 +132,19 @@ public function format(Diagnostic d) returns string[] {
     line += ": error: " + d.message;
     lines.push(line);
     Range|Position range = loc.range;
-    string[] content = loc.file.lineContent(range);
-    if content[1] == "" {
-        panic error(string `error range ${lc.toString()} is empty in ${loc.file.filename()}`);
+    File file = loc.file;
+    if file is SourceFile {
```

Comment:
This doesn't feel right, since type of the `loc.file` is `File`, not `File|SourceFile`. But If I do that, jBallerina doesn't allow me to call the methods.